### DISCUSSION TITLE
Welcome to solana-saga-unreal-sdk Discussions!

### DISCUSSION THREAD

**Discussion by etodanik** [CryptoCavemen/solana-saga-unreal-sdk#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MetaTradeCorp** [comment]
Bonjour , je suis actuellement sur le développement d'un Play2Earn mobil  '' MetaGolf'' sur Unreal Engine 5 et j'aimerais avoir des renseignement concernant l'intégration du SDK solana sur mon projet . Merci , bien cordialement .

**Reply by etodanik** [reply]
Comment puis-je vous aider ? De quelles informations auriez-vous besoin ?

**Reply by MetaTradeCorp** [reply]
Le sdk peut il gérer les récompense des joueur via des smart contract ?
